app/src/main/java/org/mifos/mobile/ui/client_charge/ClientChargeFragment.kt

```diff
@@ -17,23 +16,18 @@ import kotlinx.coroutines.launch
 import org.mifos.mobile.R
 import org.mifos.mobile.databinding.FragmentClientChargeBinding
 import org.mifos.mobile.models.Charge
-import org.mifos.mobile.ui.activities.SavingsAccountContainerActivity
 import org.mifos.mobile.ui.activities.base.BaseActivity
 import org.mifos.mobile.ui.adapters.ClientChargeAdapter
 import org.mifos.mobile.ui.enums.ChargeType
 import org.mifos.mobile.ui.fragments.base.BaseFragment
-import org.mifos.mobile.utils.ClientChargeUiState
 import org.mifos.mobile.utils.Constants
 import org.mifos.mobile.utils.Network
 import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedArrayListFromParcelable
 import org.mifos.mobile.utils.ParcelableAndSerializableUtils.getCheckedSerializable
 import org.mifos.mobile.utils.Toaster
-import org.mifos.mobile.viewModels.ClientChargeViewModel
 
-/**
- * @author Vishwajeet
- * @since 17/8/16.
- */
+/*
+
 @AndroidEntryPoint
 class ClientChargeFragment : BaseFragment() {
```

Comment:
Is this to be deleted

Reply:
Yes, as Rajan bhaiya told to keep the fragment as well, so kept it as commented code here.